packages/shared/src/components/notifications/utils.ts

```diff
@@ -60,3 +60,6 @@ const MAX_UNREAD_DISPLAY = 20;
 
 export const getUnreadText = (unread: number): string =>
   unread > MAX_UNREAD_DISPLAY ? `${MAX_UNREAD_DISPLAY}+` : unread.toString();
+
+export const getUnreadTitle = (count: number, title = ''): string =>
+  count ? `(${getUnreadText(count)}) ${title}` : '';
```

Comment:
Why do we return empty string if there's no unread notifs?

Reply:
This has evolved into something that should require the title, I think this was a mistake and needs to return the original title if it is passed.